tests/macrodata/test_pynsee_macrodata.py

```diff
@@ -133,11 +141,12 @@ def test_get_dataset_metadata_1(self):
         self.assertTrue(test2)
 
     # def test_get_dataset_metadata_2(self):
-    #     # crash download_idbank_list and test the result on get_dataset_metadata
-    #     _clean_insee_folder()
-    #     df = _get_dataset_metadata('CLIMAT-AFFAIRES')
-    #     test1 = isinstance(df, pd.DataFrame)
-    #     self.assertTrue(test1)
+    # # this takes too long
```

Comment:
Are you sure? Or is this the impact of _clean_insee_folder on other tests?

Reply:
I don't know... I tried decommenting it and it took forever, so I'm leaving it for someone else to uncomment ;) this is way outside the scope of this PR anyway